intel_extension_for_transformers/backends/neural_engine/graph/application/chat/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+#  Copyright (c) 2023 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set(TARGET chat_falcon)
+add_executable_w_warning(${TARGET} main_chat.cpp)
```

Comment:
same as quant, unify them into one cmake function

Reply:
fixed in https://github.com/intel/intel-extension-for-transformers/pull/137/commits/554e99315bf9e2fb355461a0b868841e867e58fb